### PR TITLE
[#239] Add pre game settings

### PR DESCRIPTION
**Pull request by izakfr**
Closes: #239

### CODE REVIEW

File: frontend/packages/web/src/services/http/types.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export enum HTTPActions {
   GET_ROOM = 'GET_ROOM',
   JOIN_ROOM = 'JOIN_ROOM',
   LEAVE_ROOM = 'LEAVE_ROOM',
+  UPDATE_GAME_SETTINGS = 'UPDATE_ROOM_SETTINGS',
```
Comment (benshumaker) [inline frontend/packages/web/src/services/http/types.ts:28]: This looks weird, but maybe it's fine. It is a bit strange, whether it should be called game_settings or room_settings. It's settings about a game, in a room...

**Review by benshumaker — APPROVED** [review]
Couple questions but looks good.

File: frontend/packages/web/src/services/http/types.ts (hunk 1 above)
Comment (izakfr) [reply frontend/packages/web/src/services/http/types.ts:28]: This was a mistake.

File: frontend/packages/web/src/models/room.ts [hunk 2]
```diff
@@ -3,6 +3,13 @@ export type Room = {
   name: string;
   players: { [playerID: string]: string };
   game_id: string;
+  game_settings: GameSettings;
+};
+
+export type GameSettings = {
```
[the remark this answers is not in the document]
Comment (izakfr) [reply frontend/packages/web/src/models/room.ts:9]: I'll add it later.

**Pull request closed without merging**